llumnix/backends/vllm/scheduler.py

```diff
@@ -185,8 +190,7 @@ def free_src_request(self, backend_request: SequenceGroup) -> None:
         logger.info("free seq {}".format(seq.seq_id))
         self.free_seq(seq)
 
-    @scheduler_lock
```

Comment:
you dont need lock now?

Reply:
Yes. Previously, get_instance_info is called by engine. Now, _get_instance_info is only called by schedule function in scheduler. And schedule function already have lock, so there is no need to lock _get_instance_info.